frontend/src/assets/css/material-icons.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2018-2020 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* IMPORTANT: Always use a relative path! */
+$material-font-path: "../fonts/MaterialIcons/" !default;
+
+@font-face {
+  font-family: "Material Icons";
+  font-style: normal;
+  font-weight: normal;
+  src: url("#{$material-font-path}MaterialIcons-Regular.eot"); /* For IE6-8 */
```

Comment:
I believe all the browsers that we support, support `WOFF2`. I don't think you need anything else.